src/apiary.apib

```diff
@@ -1544,6 +1544,81 @@ level `conversions` array.
                 + conversion (Conversion)
                     + conversion_type: 3 (number)
 
+
+# Group Alert
+
+Alerts are used across application to notify users about events that are triggered in the platform. Notifications are sent to users that have subscribed to an alert using a selected channel
+
+## Alert [/alerts/alert]
++ Attributes
+    + alert_id: 23 - ID of the alert
+    + code: Example -  Code used internally for reference
+    + name: Example alert - Display name of an Alert
+    + description: This alert will notify you about particular event (nullable, string) -  Fired when new campaign is created
+    + service: ExampleAlert (string) - Name of a service in DIC
+
+### Get All Available Alerts [GET]
++ Response 200 (application/json; charset=utf-8)
+    + Attributes
+        + alerts (array[Alert Wrapper])
+
+# Notifier [/alerts/notifier]
++ Attributes
+    + notifier_id: 23 - ID of a Notifier
+    + name: Email (string) - Name of the notifier
+    + service: EmailService (string) - Service registered in DIC
+
+## Get All Available Notifiers [GET]
+
++ Response 200 (application/json; charset=utf-8)
+    + Attributes
+        + notifiers(array[Notifier Wrapper])
+
+
+
+## Notification [/alerts/notification]
++ Attributes
+    + notification_id: 11l16829 - ID of the notification
+    + user_id: 3 - ID of User
+    + alert_id: 70 - ID of Alert
+    + notifier_id: 4 - ID of Notifier
+
+### Retrieve Notifications [GET]
+
+Retrieve all currently set notifications for a user
++ Response 200 (application/json; charset=utf-8)
+    + Attributes
+        + notifications (array[Notification Wrapper])
+
+### Delete User Notifications [DELETE]
+
+Delete one or more user notification
+
++ Request (application/json; charset=utf-8)
+    + Attributes
+        + notifications (array)
+            + (object)
+                + notification_id: 234 (required)
+
++ Response 200 (application/json; charset=utf-8)
+
+### Create New Notification [POST]
```

Comment:
Is there an intended update mechanism? If so, please document the PUT action too.